plugin-server/src/cdp/utils.ts

```diff
@@ -378,3 +379,66 @@ export function cyclotronJobToInvocation(job: CyclotronJob, hogFunction: HogFunc
         timings: parsedState.timings,
     }
 }
+
+const CYCLOTRON_GAUGES: Map<string, Gauge> = new Map()
+const CYCLOTRON_COUNTERS: Map<string, Counter> = new Map()
+const CYCLOTRON_HISTOGRAMS: Map<string, Histogram> = new Map()
+const CYCLOTRON_HISTOGRAM_BOUNDS: number[] = exponentialBuckets(0.5, 2, 11) // 0.5 -> 512
+
+export function emitCyclotronMetrics() {
```

Comment:
Function exists but isn't called anywhere? Tried running this locally and I'm not seeing any metrics that I would expect at http://localhost:6738/_metrics

Reply:
Yeah, sorry, I haven't actually wired it up just yet - got stuck at the labels question and figured I'd push it and get some feedback